src/backend/catalog/heap.c

```diff
@@ -284,7 +285,21 @@ static FormData_pg_attribute a8 = {
 	.attislocal = true,
 };
 
-static const FormData_pg_attribute *SysAtt[] = {&a1, &a2, &a3, &a4, &a5, &a6, &a8};
+static FormData_pg_attribute a8 = {
```

Comment:
This attribute looks same as`gp_segment_id`. Can we use `gp_segment_id` as 'gp_foreign_server' when it is a foreign table.
This can reduce the conflicts with upstream.